src/index.html

```diff
@@ -1,13 +1,289 @@
+            ₴₴ • Італійська • Піца • Американська
+          </span>
+          <span class="card__time-cooking">
+            15 - 25 Min
+          </span>
+        </div>
+
+        <div class="restaurants__item card">
+          <a href="#" class="card__link">
+            <img src="./images/photo-6.png" alt="restaurant photo" class="card__img">
+          </a>
+
+          <h2 class="card__title">
+            To Be
+          </h2>
+          <span class="card__kitchen">
+            ₴₴ • Японська • Азіатська • Суші
+          </span>
+          <span class="card__time-cooking">
+            25 - 35 Min
+          </span>
+        </div>
+
+        <div class="restaurants__item card">
+          <a href="#" class="card__link">
+            <img src="./images/photo-7.png" alt="restaurant photo" class="card__img">
+          </a>
+
+          <h2 class="card__title">
+            Druzi Cafe (вул. Прорізна)
+          </h2>
+          <span class="card__kitchen">
+            ₴₴ • Американська • Сучасна європейська
+          </span>
+          <span class="card__time-cooking">
+            30 - 40 Min
+          </span>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="footer__top">
+        <a href="#" class="footer__top--link">
+          <img src="./images/white-logo.png" alt="Logo of UberEats">
+        </a>
+      </div>
+      <div class="footer__body">
+        <div class="footer__language">
+          <select name="Lang" id="lang">
+            <option value="English" selected>
+              English
+            </option>
+            <option value="Ukrainian">
+              Ukrainian
+            </option>
+            <option value="Russian">
+              Russian
+            </option>
+          </select>
+
+          <span class="footer__body__hashtag"><span>#</span>UberEats</span>
+
+          <div class="footer__website website">
+            <a href="#" class="website-fc"></a>
+            <a href="#" class="website-tw"></a>
+            <a href="#" class="website-inst"></a>
+          </div>
+        </div>
+
+        <a href="#" class="footer__menu__item">
+          About UberEats
+        </a>
+        <a href="#" class="footer__menu__item item_row_2">
+          Become a Delivery Partner
+        </a>
+        <a href="#" class="footer__menu__item item_row_3">
+          Become a Restaurant Partner
+        </a>
+        <a href="#" class="footer__menu__item">
+          See all cities
+        </a>
+        <a href="#" class="footer__menu__item">
+          Pricing
+        </a>
+        <a href="#" class="footer__menu__item">
+          Help
+        </a>
+        <a href="#" class="footer__menu__item">
+          FAQs
+        </a>
+      </div>
+
+      <div class="footer__body--apps apps">
+        <a href="#" class="apps__link">
+          <img src="./images/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917.png" alt="AppleStore Download" class="apps__img">
```

Comment:
давай будем переносить очень длинные теги
```suggestion
          <img
            src="./images/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917.png"
            alt="AppleStore Download" class="apps__img"
          >
```